widgets/workspacemanager/workspacemanager.py

```diff
@@ -93,7 +93,8 @@ def get_workspace_to_load_path(self):
         return str(QFileDialog.getOpenFileName())
 
     def get_workspace_to_save_filepath(self):
-        path = QFileDialog.getSaveFileName()
+        extension = 'Nexus file (*.nxs)'
+        path = QFileDialog.getSaveFileName(filter=extension)
```

Comment:
On mac& windows this will force the .nxs if not found. I will now add a workaround for linux